src/lib/testutils/TestingLibraryUtils.tsx

```diff
@@ -28,14 +28,25 @@ export const createWrapper = (props?: SynapseContextType) => {
     },
   })
   const wrapperProps = props ?? MOCK_CONTEXT_VALUE
-  return function RtlWrapper({ children }: RtlWrapperProps) {
-    return (
-      <SynapseContextProvider
-        synapseContext={wrapperProps}
-        queryClient={queryClient}
-      >
-        {children}
-      </SynapseContextProvider>
-    )
+  return {
+    wrapperFn: function RtlWrapper({ children }: RtlWrapperProps) {
+      return (
+        <SynapseContextProvider
+          synapseContext={wrapperProps}
+          queryClient={queryClient}
+        >
+          {children}
+        </SynapseContextProvider>
+      )
+    },
+    queryClient: queryClient,
   }
 }
+
+/**
+ * Creates a test wrapper for components being tested with @testing-library/react. This wrapper
+ * includes context and an isolated query cache.
+ */
+export const createWrapper = (props?: SynapseContextType) => {
+  return createWrapperAndQueryClient(props).wrapperFn
+}
```

Comment:
Behavior unchanged. The new method allows retrieving the QueryClient so it can be accessed and modified in a test